content/manager_webui/default-widgets-ref.md

```diff
@@ -248,32 +255,34 @@ Any user who has access to the Secrets Store Management widget can view the valu
 
 Displays a list of snapshots of the Manager. This widget is only available to `admin` users.
 
-When restoring snapshots, a specific process must be followed, relating to whether you want to use your existing VM or create a new one. For more information, [click here]({{< relref "manager/upgrade.md" >}}).
-
 ![snapshots-list]({{< img "ui/widgets/snapshots-list.png" >}})
 
+Parameters presented below, on the screenshot, can be provided during snapshot creation. 
+
+![snapshots-list]({{< img "ui/widgets/snapshots-creation-modal.png" >}})
+
+When restoring snapshots, a specific process must be followed, relating to whether you want to use your existing VM or create a new one. For more information, [click here]({{< relref "manager/upgrade.md" >}}).
```

Comment:
When you [restore snapshots]({{< relref "cli/snapshots.md" >}}),  use the process for either your existing VM or a new VM.
(I don't think I understood this correctly.

Reply:
That's not my sentence. I just copied it. I'm not familiar with those mechanism, but your sentence sounds good.